### ISSUE TITLE
PIA provider failing to download config files

### ISSUE THREAD

**Issue by glitchcrab** [haugene/docker-transmission-openvpn#1559]
**Describe the problem**

Using `3.2`, the container fails to start (it was working previously with the same image/config). Narrowed the issue down to this line: https://github.com/haugene/docker-transmission-openvpn/blob/master/openvpn/pia/configure-openvpn.sh#L27

I can't reproduce it outside of the container though - curling the file and then unzipping it with the commands used in `configure-openvpn.sh` works just fine.

**Add your docker run command**
Container is managed by an Ansible playbook:

```yaml
  - name: transmission
    docker_container:
      name: transmission
      image: "{{ docker['transmission']['image'] }}:{{ docker['transmission']['tag'] }}"
      pull: false
      state: started
      restart_policy: unless-stopped
      memory: 500M
      capabilities:
        - net_admin
      dns_servers:
        - 9.9.9.9
        - 1.1.1.1
      published_ports:
        - "10.101.0.50:9091:9091"
      volumes:
        - "/srv/docker/transmission/data:/data"
        - "/srv/docker/transmission/scripts:/scripts"
        - "/srv/store/downloads/watch:/watch"
        - "/srv/store/downloads:/downloads"
        - "/srv/store/downloads/incomplete:/data/incomplete:rw"
        - "/etc/localtime:/etc/localtime:ro"
      env:
        CREATE_TUN_DEVICE: "true"
        OPENVPN_PROVIDER: "PIA"
        OPENVPN_CONFIG: "CA Montreal"
        OPENVPN_USERNAME: "{{ transmission['pia']['username'] }}"
        OPENVPN_PASSWORD: "{{ transmission['pia']['password'] }}"
        WEBPROXY_ENABLED: "false"
        LOCAL_NETWORK: "10.101.0.0/21"
        TRANSMISSION_ALT_SPEED_DOWN: "7000"
        TRANSMISSION_ALT_SPEED_UP: "1500"
        TRANSMISSION_ALT_SPEED_ENABLED: "true"
        TRANSMISSION_ALT_SPEED_TIME_ENABLED: "true"
        TRANSMISSION_ALT_SPEED_TIME_BEGIN: "540"
        TRANSMISSION_ALT_SPEED_TIME_END: "1020"
        TRANSMISSION_ALT_SPEED_TIME_DAY: "62"
        TRANSMISSION_DOWNLOAD_LIMIT: "12500"
        TRANSMISSION_DOWNLOAD_LIMIT_ENABLED: "true"
        TRANSMISSION_UPLOAD_LIMIT: "3000"
        TRANSMISSION_UPLOAD_LIMIT_ENABLED: "true"
        TRANSMISSION_DHT_ENABLED: "false"
        TRANSMISSION_UTP_ENABLED: "false"
        TRANSMISSION_DOWNLOAD_QUEUE_ENABLED: "false"
        TRANSMISSION_ENCRYPTION: "2"
        TRANSMISSION_PEER_LIMIT_GLOBAL: "500"
        TRANSMISSION_PEER_LIMIT_PER_TORRENT: "50"
        TRANSMISSION_RPC_BIND_ADDRESS: "0.0.0.0"
        TRANSMISSION_WATCH_DIR_ENABLED: "false"
        DOCKER_LOG: "true"
        OPENVPN_OPTS: "--inactive 3600 --ping 10 --ping-exit 60 --pull-filter ignore ping"
```

**Logs**
```
Starting container with revision: baed33520630e88ba34842cab05f2cd3fd6cf28f
Creating TUN device /dev/net/tun
Using OpenVPN provider: PIA
Provider PIA has a custom setup script, executing it
Downloading OpenVPN config bundle openvpn-nextgen into temporary file /tmp/tmp.IcpffI
Extract OpenVPN config bundle into PIA directory /etc/openvpn/pia
unzip: short read
```

**Host system:**
Ubuntu, running `Docker version 19.03.5, build 633a0ea838`

**Comment by haugene** [comment]
Yes, the script you're referring to (on master) is working. But the one in 3.2 is not. PIA changed the links to their configs. See #1548
You can either pull and run the latest image, or set PIA_OPENVPN_CONFIG_BUNDLE=openvpn which will fix it for 3.2 as well. It used to be called openvpn-nextgen, but after the migration they removed -nextgen. That is the new default in the newer builds.

**Comment by glitchcrab** [comment]
I should have guessed that this was caused by PIA in some way :roll_eyes: thanks for the explanation!
